tasks/axe-webdriver.js

```diff
@@ -0,0 +1,58 @@
+/*! aXe-grunt-webdriver
```

Comment:
ok, I could pull it out into a separate module that has a dependency on selenium-webdriver. This would solve both problems.